pages/en/lb3/Defining-middleware.md

```diff
@@ -690,17 +690,10 @@ Instead, it will be ignored because the route was already matched.
 
 ### Error-handling middleware
 
-Use error-handling middleware to deal with request errors.
+Use error-handling middleware to deal with request errors. The  [`strong-error-handler`](https://github.com/strongloop/strong-error-handler/) middleware is scaffolded by default and sufficient for many projects, but additional custom error-handling middleware may be registered as necessary.
 While you are free to register any number of error-handling middleware, be sure to register them in the "final" phase.
-LoopBack registers two error-handling middleware by default:
 
-* `urlNotFound` middleware converts all requests that reach the middleware into an Error object with status 404,
-  so that 404 error responses are consistent with "usual" error responses.
```

Comment:
TYVM @bajtos. I agree, sounds like we need to move this into a separate Issue/discussion. I'm not convinced that it's imperative for strong-remoting to defend its route space, but I understand your logic and see that this is getting out of scope for this PR.

> We don't have any plans for that, but now that you mentioned it, I think it may make sense to provide this useful middleware in a place where other users can reuse it and strong-error-handler looks like the perfect vehicle for that!

I am attracted to the consistency of this.

> As for this pull request, I let's keep the mention of urlNotFound middleware as it is now.

Agreed. @crandmck - I'll update this shortly to bring this back and then this should be ready to merge.